src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -0,0 +1,197 @@
+package au.org.emii.geoserver.wms;
+
+import org.apache.commons.io.IOUtils;
+import org.dom4j.Document;
+import org.dom4j.DocumentException;
+import org.dom4j.io.SAXReader;
+import org.dom4j.tree.DefaultText;
+import org.dom4j.xpath.DefaultXPath;
+import org.geotools.util.logging.Logging;
+import org.json.JSONObject;
+
+import java.io.*;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import java.net.URL;
+import java.util.*;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class Ncwms {
+    static Logger LOGGER = Logging.getLogger("au.org.emii.geoserver.wms.ncwms");
```

Comment:
`getLogger()` can take a class instead of a String. I think this is preferred as it means the logger name can't get out of sync with the class (It's enforced by the compiler, essentially).